users/d4mation/rules.mk

```
@@ -0,0 +1,15 @@
+SRC += d4mation.c
+
+BOOTMAGIC_ENABLE = no
+EXTRAFLAGS += -flto
```

You can do this: 
```suggestion
LTO_ENABLE = yes
```

That enables LTO, and disableds action macros and action functions.